.github/workflows/tests.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+name: 'API-Tests'
+
+on:
+  push:
+    branches:
+      - "greggid/ci-cd-tests"
```

Comment:
Are you sure what it test needs to be triggered only by pushing to your branch?

Reply:
done